Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3244,6 +3244,43 @@ - (BOOL)_hasVisibleColorExtensionView:(WebCore::BoxSide)side
     return view && ![view isHiddenOrFadingOut];
 }
 
+- (void)_addReasonToHideTopContentInsetFill:(WebKit::HideContentInsetFillReason)reason
+{
+    if (_reasonsToHideTopContentInsetFill.contains(reason))
+        return;
+
+    if (_reasonsToHideTopContentInsetFill.isEmpty())
```

Comment:
You could also use `operator bool()` here, like you did in `_removeReasonToHideTopContentInsetFill`?

Reply:
Good point!